
2023-01-31 16:46:09.085489: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /cm/shared/apps/slurm/current/lib64/slurm:/cm/shared/apps/slurm/current/lib64
2023-01-31 16:46:09.085711: E tensorflow/stream_executor/cuda/cuda_driver.cc:318] failed call to cuInit: UNKNOWN ERROR (303)
2023-01-31 16:46:09.085765: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (c-0-17): /proc/driver/nvidia/version does not exist
2023-01-31 16:46:09.086080: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2023-01-31 16:46:09.106217: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2197325000 Hz
2023-01-31 16:46:09.106370: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x555559179c20 executing computations on platform Host. Devices:
2023-01-31 16:46:09.106384: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version
Using TensorFlow backend.

SynthSeg-robust 2.0

using 1 thread
predicting 1/1

the following problem occured with image /mnt/isilon/bgdlab_resnas03/Data/16p/rawdata/sub-14708/ses-0010/anat/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w.nii.gz :
Traceback (most recent call last):
  File "/mnt/isilon/bgdlab_resnas03/code/SynthSeg-lib/SynthSeg/predict_synthseg.py", line 167, in predict
    path_resample=path_resampled[i])
  File "/mnt/isilon/bgdlab_resnas03/code/SynthSeg-lib/SynthSeg/predict_synthseg.py", line 428, in preprocess
    raise Exception('input should have 3 dimensions, had %s' % n_dims)
Exception: input should have 3 dimensions, had 2

resuming program execution


segmentation  saved in:    /mnt/isilon/bgdlab_resnas03/Data/16p/derivatives/synthseg+/sub-14708/ses-0010/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w_synthseg.nii.gz
posteriors saved in:       /mnt/isilon/bgdlab_resnas03/Data/16p/derivatives/synthseg+/sub-14708/ses-0010/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w/posterior_probability_maps.nii.gz
volumes saved in:          /mnt/isilon/bgdlab_resnas03/Data/16p/derivatives/synthseg+/sub-14708/ses-0010/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w/volumes.csv
QC scores saved in:        /mnt/isilon/bgdlab_resnas03/Data/16p/derivatives/synthseg+/sub-14708/ses-0010/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w/qc_scores.csv

If you use the new robust version of SynthSeg in a publication, please cite:
Robust Segmentation of Brain MRI in the Wild with Hierarchical CNNs and no Retraining
B. Billot, M. Collin, S.E. Arnold, S. Das, J.E. Iglesias

ERROR: some problems occured for the following inputs (see corresponding errors above):
/mnt/isilon/bgdlab_resnas03/Data/16p/rawdata/sub-14708/ses-0010/anat/sub-14708_ses-0010_acq-TSEVARIANTDim3Size_run-0005_T1w.nii.gz

real	0m18.701s
user	0m7.470s
sys	0m1.346s
Job finished running!
